app/services/notifications.rb

```diff
@@ -29,4 +29,12 @@ def self.comment_data(comment)
       }
     }
   end
+
+  def self.moderator_availability_delay
+    28.hours
+  end
+
+  def self.available_moderators
+    User.with_role(:trusted).where("last_moderation_notification < ?", moderator_availability_delay.ago)
```

Comment:
I'd move `available_moderators` this inside the module `Notifications::Moderation`, not `Notifications`

Reply:
Agree, the method should be as private, as possible. We don't need moderators in most of the `Notifications`